saleor/graphql/order/tests/test_filters.py

```diff
@@ -0,0 +1,108 @@
+import pytest
+
+from ....order.models import Order
+from ..filters import filter_customer
```

Comment:
I would recommend to do `@mock.patch("..._filter_customer_by_email_first_or_last_name", wraps=_filter_customer_by_email_first_or_last_name)`. (example for tests when you're looking for first or last name). 
Then `assert mocked.called`. `wraps` will execute `_filter_customer_by_email_first_or_last_name`, but you will be also able to fully confirm that your expected logic was called.
Thanks to this we will be sure that we always trigger a proper search logic.